app/views/appliance_comments/_compact_form.html.haml

```diff
@@ -8,7 +8,7 @@
 
   .field
     = f.label :category
-    = collection_select(:appliance_comment, :category, ApplianceComment.categories.keys ,:to_s,:to_s, prompt: true)
+    = collection_select(:appliance_comment, :category, f.object.available_actions ,:to_s,:to_s, prompt: true)
```

Comment:
The value of the select options will be a string, not an id. I found no viable way to do this with `collection_select`, but with `options_for_select` it would be easier (see:  #30 ).

Reply:
Why is it a problem that the value is a string? Rails enum handles the conversion.
The intention of the enum to hide the underlying ids so if would be a step backward I'think.